pkg/cache/v2/status.go

```diff
@@ -77,11 +90,24 @@ type ResponseWatch struct {
 	Response chan Response
 }
 
+// DeltaResponseWatch is a watch record keeping both the delta request and an open channel for the delta response.
+type DeltaResponseWatch struct {
+	// Request is the original delta request for the watch
```

Comment:
What does original mean in the context of delta? First of the stream? Or just the latest one since we recreated watches for each request?

Reply:
Responded above to this, I'll change the comment!